cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/counters/ScopeList.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.cradle.cassandra.counters;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+
+public class ScopeList {
+    private final HashSet<String> scopeSet;
```

Comment:
Why do you need the ceratin implementation here? Maybe you can use `Set` interface instead?